neonvm/config/default-vxlan/network/overlay-for-pods.yaml

```diff
@@ -1,17 +1,18 @@
 apiVersion: "k8s.cni.cncf.io/v1"
 kind: NetworkAttachmentDefinition
 metadata:
-  name: overlay-net-ipam
+  name: overlay-for-pods
   namespace: system
 spec:
   config: '{
     "cniVersion": "0.3.1",
-    "name": "neonvm-overlay-net-ipam",
+    "name": "overlay-for-pods",
     "type": "bridge",
     "bridge": "neon-br0",
     "ipam": {
       "type": "whereabouts",
-      "range": "10.100.0.0/20",
-      "range_start": "10.100.0.100"
-      }
-    }'
+      "range": "10.100.0.0/16",
+      "range_start": "10.100.1.0",
+      "range_end": "10.100.127.255"
+    }
+  }'
```

Comment:
the same syntax as in [Whereabouts CNI examples](https://github.com/k8snetworkplumbingwg/whereabouts#an-example-configuration-using-a-networkattachmentdefinition), I think it's ok.

Reply:
Ah, actually my mistake - this is currently what `main` does. It can be left for a separate PR either way.

---

To clarify, what I mean is this: `main` currently produces `neonvm.yaml` that has the following entry:

```
apiVersion: k8s.cni.cncf.io/v1
kind: NetworkAttachmentDefinition
metadata:
  name: neonvm-overlay-net-ipam
  namespace: neonvm-system
spec:
  config: '{ "cniVersion": "0.3.1", "name": "neonvm-overlay-net-ipam", "type": "bridge",
    "bridge": "neon-br0", "ipam": { "type": "whereabouts", "range": "10.100.0.0/20",
    "range_start": "10.100.0.100" } }'
```
**Note:** The line breaks that you see are exactly what was inserted by kustomize.

This means it's really annoying to make changes to the config without re-running kustomize.

That was the reasoning behind #300 — because we have some differences in the autoscaler-agent config between dev/staging/prod (billing settings), and `|` means the line breaks are preserved, whereas with quotes it does does this weird thing.

Reply:
I prefer leave these NetworkAttachmentDefinitions as is right now (as it tested in different environments). May be raise another PR with improvement this part.